packages/@okta/vuepress-site/docs/reference/system-log-query/index.md

```diff
@@ -0,0 +1,510 @@
+| trsf8nlpDJZTZeFlcc8nszbjw   | Wij-7a4YuniRd9yTmWHqqAAAAKY   | 5526a4c4-7f68-4b2a-bab7-2d10ebaeeb1c   | mim.checkOSXAccessEligibility.true            | *blank*                               |
+|                             | Wij-764YuniRd9yTmWHrkAAAAGw   | 232774ba-8feb-4b00-a732-e0ec99a24434   | user.session.start                            | User signs in to Okta                    |
+| trswPONv4wIRaKDNWVVcmtceg   | Wij-6K4YuniRd9yTmWHo9wAAAAY   | d31d819a-1427-45b0-a8b4-8a8fb40c72f1   | user.session.start                            | User sign in to Okta                    |
+|                             | Wij-564YuniRd9yTmWHoaQAAAII   | 0cc6f4c8-9b91-4a70-b5c4-09d6ad159d32   | mim.checkOSXAccessEligibility.true            | *blank*                               |
+|                             | Wij-2q4YuniRd9yTmWHjRAAAADA   | 92606da8-7eeb-4ad7-8ffb-502dd0ec64cc   | user.authentication.sso                       | User SSOs to the app            |
+| *null*                      | Wm@-R2s5lEMbNIB03krtvAAACyo   | 566671be-ec0b-400d-ad2e-6fc73ed12fb1   | user.session.start                            | User signs in to Okta                    |
+[[.table-word-break]]
+
+As evidenced by the `null` `authenticationContext.externalSessionId` field in the last row, neither `transaction.id` nor `uuid` maintain a many-to-one relationship with `authenticationContext.externalSessionId`. In this particular case, the `null` `authenticationContext.externalSessionId` is a user sign-in failure. There’s no session granted to the user's client because of the sign-in failure.
+
+## List events
+
+Use the [List all System Log Events](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/SystemLog/#tag/SystemLog/operation/listLogEvents) endpoint to retrieve a list of events from your org. By default, this request returns a limit of 100 response objects.
+
+##### Request types
+
+All requests to the `/api/v1/logs` endpoint fall into one of these two categories:
+
+* [Polling Requests](#polling-requests)
+* [Bounded Requests](#bounded-requests)
+
+###### Polling requests
+
+Polling requests are for situations when you want to consume an ongoing stream of events from Okta.
+
+Example use cases include:
+
+* [Ingesting System Log data into an external SIEM system](#transfer-data-to-a-separate-system).
+* Using System Log data for real-time monitoring.
+
+For a request to be a polling request, it must meet the following request parameter criteria:
+
+* `until` is unspecified.
+* `sortOrder` is `ASCENDING`.
+
+Polling requests to the `/api/v1/logs` API have the following semantics:
+
+* They return every event that occurs in your org.
+* The returned events are time filtered by their internal "persistence time" to avoid skipping records due to system delays (unlike [Bounded Requests](#bounded-requests)).
```

Comment:
Update for clarity:
The returned events are ordered by the internal "persistence time" which reflects when an event is actually committed to the log, and not necessarily the timestamp when the event occurred. The persistence time may differ from the externally available `published` timestamp.